source-hive/src/main/java/org/apache/kylin/source/hive/BeelineHiveClient.java

```diff
@@ -272,6 +261,9 @@ private void extractTableParam(ResultSet resultSet, HiveTableMetaBuilder builder
             if ("numFiles".equals(resultSet.getString(2).trim())) {
                 builder.setFileNum(Long.parseLong(resultSet.getString(3).trim()));
             }
+            if ("numRows".equals(resultSet.getString(2).trim())) {
+                builder.setRowNum(Long.parseLong(resultSet.getString(3).trim()));
```

Comment:
Is it certain that "numRows" will always be gotten in hive table's meta? As I know, there is possibility that it be -1